test/nni_test/nnitest/validators.py

```diff
@@ -93,7 +93,7 @@ class NnicliValidator(ITValidator):
     def __call__(self, rest_endpoint, experiment_dir, nni_source_dir, **kwargs):
         print(rest_endpoint)
         exp = Experiment()
-        exp.connect_experiment(rest_endpoint)
+        exp.connect_experiment(int(rest_endpoint.split(':')[-1]))
```

Comment:
Is this the correct way to get the port part? What if the endpoint ends with `/`?

Reply:
it's a workaround, and will refactor in next pr, this will never be called right now.